### DISCUSSION TITLE
Size mismatch error with Pytorch's TorchEntityRecognizer on evaluation

### DISCUSSION THREAD

**Discussion by felipefoschiera** [explosion/spaCy#9497] [Help: Coding & Implementations] [answered]
I am having some trouble evaluating a transformer model (`roberta-base`) trained with the torch_ner pipeline.
I was able to run both train+evaluation commands with `spaCy's ner + transformer`, as well as `torch_ner + vectors`. But for `torch_ner + transformer`, I was only able to train, and I get the following error when trying to evaluate the model:

```
RuntimeError: Error(s) in loading state_dict for TorchEntityRecognizer:
        size mismatch for model.input_layer.weight: copying a param with shape torch.Size([64, 768]) from checkpoint, the shape in current model is torch.Size([64, 1]).
```

The command executed was
`python -m spacy evaluate training/training_pytorch_trf/model-best corpus/test.spacy --output training/training_pytorch_trf/metrics.json --code scripts/custom_functions.py --gpu-id 0`

The `config.cfg` file I am using is attached to this discussion.
[config_pytorch_trf_txt.txt](https://github.com/explosion/spaCy/files/7366807/config_pytorch_trf_txt.txt)

**Comment by svlandeg** [comment, marked as answer]
Hi! Thanks for the report and sorry for the late follow-up. We're working on a fix: https://github.com/explosion/projects/pull/80
